packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -59,62 +60,107 @@ export const ValueInput = styled( NumberControl )`
 	}
 `;
 
-const baseUnitLabelStyles = css`
-	appearance: none;
-	background: transparent;
-	border-radius: 2px;
-	border: none;
-	box-sizing: border-box;
-	color: ${ COLORS.darkGray[ 500 ] };
-	display: block;
-	font-size: 8px;
-	letter-spacing: -0.5px;
-	outline: none;
-	padding: 2px 1px;
-	text-align-last: center;
-	text-transform: uppercase;
-	width: 20px;
-
-	${ rtl( { borderTopLeftRadius: 0, borderBottomLeftRadius: 0 } )() }
-`;
+const baseUnitLabelStyles = ( { selectSize }: SelectProps ) => {
+	const sizes = {
+		default: css`
+			box-sizing: border-box;
+			padding: 2px 1px;
+			width: 20px;
+			color: ${ COLORS.darkGray[ 500 ] };
+			font-size: 8px;
+			letter-spacing: -0.5px;
+			text-transform: uppercase;
+			text-align-last: center;
+		`,
+		large: css`
+			box-sizing: border-box;
+			min-width: 24px;
```

Comment:
I still don't have a clear idea of when we should use the `space` utility and when we should just write "raw" px values. What do you think?

Reply:
😂 

I think my current rule of thumb is:

- If it's less than 4px, use the exact value
- If it's size variant value, use the exact value (for readability, and because there's a 30px variant!)
- If it's related to spacing (margin/padding), prefer to use `space`

Assuming the main point of `space` is to encourage people to use multiples of 4px, it shouldn't _really_ matter whether the `space` util is used 🤞

Reply:
Sounds reasonable